debian/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:jessie
+RUN apt-get update && apt-get install -qqy curl python2.7 gcc python-dev python-setuptools apt-transport-https lsb-release  && \
```

Comment:
for consistency with the generic installer here:
[https://cloud.google.com/sdk/downloads#apt-get](https://cloud.google.com/sdk/downloads#apt-get)